gateway/1.5/modules/ROOT/pages/policies-included-traffic-management.adoc

```diff
@@ -0,0 +1,155 @@
+= Traffic Management for Multiple Upstream Services Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route weight, traffic management, multiple upstream services
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Traffic Management for Multiple Upstream Services
+>s|Summary      | Manages API instance traffic to multiple upstream services from a single consumer endpoint
+>s| First Flex Gateway version available | v1.5.0
+.4+>.^s| Returned Status Codes | No return codes exist for this policy.
+[%autowidth.spread,cols="a,a"]
+|
+|===
+
+[NOTE]
+====
+The following document applies only to Flex Gateway running in Local Mode. To configure multiple upstream services for Flex Gateway running in Connected Mode, see xref:api-manager::create-instance-task-flex.adoc#traffic-management[].
+====
+
+== Summary
+
+Flex Gateway running in Local Mode supports API instances that expose multiple upstream services through a single consumer endpoint.
+
+Flex Gateway manages traffic by using different routes with defined rulesets and route order to direct traffic to different sets of upstream services. Each API instance supports multiple routes that can each direct traffic to multiple upstream services.
+
+API instances with multiple upstream services in Local Mode are configured by using `PolicyBinding` YAML configuration resources. Each `route-weighted` resource defines a single a route. To add additional routes, you can add multiple `route-weighted` resources.
+
+For a tutorial about how to publish an API instance with this policy applied, see xref:flex-local-publish-api-multiple-services.adoc[].
+
+NOTE: The Traffic Management for Multiple Upstream Services Policy does not affect the xref:policies-included-tls-outbound[Outbound Transport Layer Security Policy]. You can still apply an Outbound TLS policy to each upstream service.
+
+== Configuring Policy Parameters
+
+Refer to the following policy definition and table of parameters:
+
+----
+policyRef:
```

Comment:
For consistency with other policy pages, this policy definition should specify data types and whether or not the parameter is required or optional. The definition shouldn't include example values.

Please see:
https://dev-docs-internal.kqa.msap.io/beta-flex-internal/gateway/1.5/policies-included-tls#configuring-policy-parameters